app/models/drupal_users.rb

```diff
@@ -29,7 +29,7 @@ def internalShouldIndexSolr
   end
 
   def user
-    User.find_by_username name
+    User.where("username collate latin1_swedish_ci = '" + name + "'")
```

Comment:
Whoa, this is intense. do we need to do this elsewhere? Do we need to limit character set allowed?